step-capstone/src/components/Finalized.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {useEffect, useState, useRef} from 'react';
+import ReactDOM from 'react-dom';
+import TravelObject from './TravelObject';
+import TimeLine from './TimeLine';
+export default function Finalized(props) {
+    const hashMap = new Map();
+    let prevDiv = useRef([]);
+    const [displayDate, setDisplayDate] = useState(props.startDate);
+
+    const handleDisplayDateChange = (newDisplayDate) => {
+        setDisplayDate(newDisplayDate);
+    }
+
+    const sameDate = (timeA, timeB) => {
+      return (timeA.getDate() === timeB.getDate() && timeA.getMonth() === timeB.getMonth() && timeA.getFullYear() === timeB.getFullYear());
+    }
+    const pushItemToArrayInHashMap = (item, dateKey) => {
```

Comment:
not super pressing but the array in hashmap naming is not super helpful. i want to know what the hashmap and array are in relation to the code (hashmap is our dates, array is the list of items on the date) as I already know by their declaration what data structure they are. could you try renaming the hashmap, array and related and methods for clarity.

Reply:
This method could be named "addItemToDate()"

Reply:
Good suggestions! I will make sure to change the names of the method and data structures!